R/FilterStateChoices.R

```diff
@@ -15,6 +16,66 @@
 #' isolate(filter_state$set_selected("B"))
 #' isolate(filter_state$set_keep_na(TRUE))
 #' isolate(filter_state$get_call())
+#'
+#' \dontrun{
+#' # working filter in an app
+#' library(shiny)
+#'
+#' data_choices <- c(letters[1:4], NA)
+#' filter_state_choices <- ChoicesFilterState$new(
+#'   x = data_choices,
```

Comment:
set `x` as a vector with repeated levels for meaningful counts
```suggestion
#'   x = sample(data_choices, 100, replace = TRUE),
```

Reply:
:+1: